Header header
string Barcode_name
uint32 Short_name
float32 AvgX
float32 AvgY
float32 Avgangle
